stix_shifter_modules/msatp/stix_translation/README.md

```diff
@@ -1,4 +1,4 @@
-# MSATP UDS Connector
+# Microsoft Defender for Endpoint(previously Microsoft Defender ATP) UDS Connector
```

Comment:
Add a space at the start of the opening parenthesis.

Reply:
done